jc/parsers/systemextensionsctl.py

```diff
@@ -0,0 +1,143 @@
+"""jc - JSON Convert `systemextensionsctl list` command output parser
+
+Usage:
+
+    $ systemextensionsctl list | jc --systemextensionsctl-list
```

Comment:
I think the parser name may not be correct here. `--systemextensionsctl-list` is a little long, too. I think `--systemextensionsctl` will work.